src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1723,11 +1785,21 @@
 helps['network vnet show'] = """
     type: command
     short-summary: Get the details of a virtual network.
+    long-summary: To learn more about virtual networks and settings, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-network#a-name--view-vnetaview-virtual-networks-and-settings
+    examples:
+        - name: Typical usage
+          text: >
+            az network vnet show -n myVNet -g MyResourceGroup
 """
 
 helps['network vnet update'] = """
     type: command
     short-summary: Update a virtual network.
+    long-summary: To learn more about managing virtual networks, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-network
+    examples:
```

Comment:
Remove, or improve. Potential improvement is to show how to link an existing vnet to a set of DNS resolution servers across multiple Azure regions.